### ISSUE TITLE
is it possible to have colors with npm wildcards command?

### ISSUE THREAD

**Issue by anisg** [open-cli-tools/concurrently#234]
Is it possible to add colors to npm wildcards commands?

I can have colors without using wildcards, eg:
```
concurrently --names 'api,front' -c 'bgBlue.bold,bgMagenta.bold' 'echo api' 'echo front' 
```

But when I use npm wildcards there is no colors: e.g in package.json
```
"scripts": {
    "dev-api": "echo api",
    "dev-front": "echo front",
    "dev": "concurrently  -c 'bgBlue.bold,bgMagenta.bold' npm:dev-*"
  },
```

**Comment by milahu** [comment]
duplicate of #210

**Comment by gustavohenke** [comment]
Fix out in v6.3.0.
